### DISCUSSION TITLE
Automatically blacklist sites with custom dark mode enabled

### DISCUSSION THREAD

**Discussion by PianoMastR64** [darkreader/darkreader#7875] [Ideas]
Dark Reader should detect if a website is being displayed in its own custom dark mode and automatically blacklist it with a warning that it has done so. Also, some websites have specific pages that are dark-themed and other pages that are light, so black/whitelisting domains isn't granular enough for this suggestion. Google for example does this a lot.

It would also be cool if it kept a crowdsourced database of all websites that have a custom dark mode option somewhere and let the user know about it.

These suggestions would make me more comfortable just letting this extension handle dark mode for all websites with a blacklist so I won't fear missing out on any site's intended dark theme, which usually looks better in my opinion.

**Comment by Gusted** [comment]
See #2774
